test/reducers/user_revisions.spec.js

```diff
@@ -0,0 +1,27 @@
+import deepFreeze from 'deep-freeze';
+import '../testHelper';
+import user_revisions from '../../app/assets/javascripts/reducers/user_revisions';
```

Comment:
As a convention and to match how it's used in the app, this import should match the name of the imported function in the other file: `userRevisions`.